src/CommandingOfficers/Tech.java

```diff
@@ -552,7 +552,7 @@ private UnitModel createBattleMechModel()
     UnitModel BattleMech = mdTank.clone();
     BattleMech.name = "BattleMech";
     BattleMech.role = BattleMech.role | UnitModel.SURFACE_TO_AIR;
-    BattleMech.costShift = mdTank.getCost() + (antiAir.getCost()/2);
+    BattleMech.costBase = mdTank.costBase*2 + (antiAir.costBase/2);
```

Comment:
x2?

Reply:
Before it was `costShift`, which added onto the original Md Tank cost.
This is setting `costBase`, so I included the cost of the original Md Tank.

If you don't want that, okay. I was just keeping the semantics the same.